### DISCUSSION TITLE
Loose control when i press "play"

### DISCUSSION THREAD

**Discussion by djnekkid** [Courseplay/Courseplay_FS22#2564] [General]
Hi all.

Im not sure if this is a known bug or not, or anyone else are haveing the same problem(s).

Occationally, when I press "play" on a course, i "loose control" of the game. The vehicle just drive ahead a little bit, but the camera freeze.

I can still go into the menus and such, but even if i try to save the game, it dont save.
Only thing i can do, is to quit back to the menus, and reload the save. And we all know how "dumb" the autosave system is.

I've only encountered it on combine unloaders and bale collectors, but i guess thats the type of thing i press "play" on more then others, so it might be a sample size thing :)

-DJ

**Comment by Tensuko** [comment]
Sounds like a LUA Error.
Need your logfile when this happens.
https://github.com/Courseplay/Courseplay_FS22/wiki

**Comment by djnekkid** [comment]
https://drive.google.com/file/d/1GEeHStXW9TLho-wKLuW1hNhThbEUTAvd/view?usp=sharing

here is the logfile.

**Comment by djnekkid** [comment]
Just did a little test.
It seems to happen when i have alot of CP and AD vehicles going.
It dont seem to happen to "tracked" vehicles
It does seem to happen to "unloader" and "collectors".

**Comment by Tensuko** [comment]
Error in log:
2023-05-29 20:55 :02 [info lp116118] Scanning field 1 on Western Australia, prefer custom fields false
2023-05-29 20:55 Error: Running LUA method 'update'.
C:/Users/Thomas/Documents/My Games/FarmingSimulator2022/mods/FS22_Courseplay/scripts/ai/AIDriveStrategyFieldWorkCourse.lua:683: attempt to index a nil value

Not sure why and how this happens.

**Reply by djnekkid** [reply]
it happens in both Field1 and 2, altho Field1 are combined from 2 fields of similar size

**Comment by Tensuko** [comment]
Should be fixed in 7.2.1.7
